tests/lib/testing_classes.py

```diff
@@ -10,7 +10,7 @@
 import platform
 
 from tests.lib.db_handler import DBHandler
-from tests.lib.file_handler import RemoteFileHandler
+from distribution.file_handler import RemoteFileHandler
```

Comment:
I agree with @iLLiCiTiT about the module name and the question is if this shouldn't live in openpype module?

Reply:
I don't think it good idea, I think importing it from openpype might be problematic.